action/clihelper.go

```diff
@@ -192,6 +192,66 @@ func (s *Action) askForPrivateKey(prompt string) (string, error) {
 	return "", fmt.Errorf("no valid user input")
 }
 
+// askForGitConfigUser will iterate over GPG private key identities and prompt
+// the user for selecting one identity's name and email address each for being used as
+// git config user.name and git config user.email, respectively.
+// On error or no selection, name and email will be empty.
+// If s.isTerm is false (i.e., the user cannot be prompted), however,
+// the first identity's name/email pair found is returned.
+func (s *Action) askForGitConfigUser() (string, string, error) {
+	var (
+		name       string
+		email      string
+		useCurrent bool
+	)
+
+	keyList, err := s.gpg.ListPrivateKeys()
+	if err != nil {
+		return "", "", err
+	}
+	keyList = keyList.UseableKeys()
+	if len(keyList) < 1 {
+		return "", "", fmt.Errorf("No usable private keys found")
+	}
+
+	for _, key := range keyList {
+		for _, identity := range key.Identities {
+			useCurrent = false
+
+			if s.isTerm {
```

Comment:
I thought this _might_ be an (although not very likely) scenario but it definitely simplifies things to not split identities up, agreed.